go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/eclipse-kanto/azure-connector
 
-go 1.17
+go 1.19
```

Comment:
Do we need this change? That's the minimum Go version to build the component. As soon as there are no 1.18/1.19 features. It can stay.